examples/step-3/doc/results.dox

```diff
@@ -120,15 +120,12 @@ suggestions:
   (here 1e-12) of -1 and 1. Try this immediately after calling
   GridGenerator::hyper_cube(), as before:
   @code
-  for (const auto &cell : dof_handler.active_cell_iterators())
-    {
-        for (auto &face : cell->face_iterators())
-          if ((std::fabs(face->center()(1) - (-1.0)) < 1e-12) ||
-              (std::fabs(face->center()(1) - (1.0)) < 1e-12))
-            face->set_boundary_id(1);
-    }
+  for (auto &face : triangulation.active_face_iterators())
+      if (std::fabs(face->center()(1) - (-1.0)) < 1e-12 || 
+          std::fabs(face->center()(1) - (1.0)) < 1e-12)
+          face->set_boundary_id(1);
```

Comment:
This indentation is a little confusing. At first sight, I would think that `face->set_boundary_id(1)` is part of the `if`-statement. Would you mind aligning with it with the `r` in `for` (just as before)?

Reply:
I have made a new commit. Please check if that works?

Reply:
The original code was already weirdly indented. Please remove the empty line again and use consistent two-space indentation. This should move the `face->set_boundary_id(1);` under the space after the `if`, and so to the *left* of the second `std::fabs`. This should make it more readable.

Reply:
Yes. I think I have understood the changes you requested, and have fixed this just now.

Reply:
Question: Does `clang-format` work for `doxygen` code embedded inside such c++ comment blocks?  I am asking because I think this will make things easier by avoiding the need to indent by hand.

Reply:
No.

Reply:
Okay, how do the developers format the code embedded within doxygen comments?  For longer blocks of code, this seems painful to do the indentations by hand?

Reply:
By hand, or by placing the code into a program that the indent script runs on and the copying it back out.